packages/py-api/py_api/controllers/teams_controller.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from bson.errors import InvalidId
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
+
+
+class TeamsController:
+
+    def fetch_teams() -> JSONResponse:
+        teams = list(t_col.find())
+
+        if not teams:
+            return JSONResponse(content={"message": "No teams were found in db"}, status_code=404)
+
+        return JSONResponse(content={"teams": json.loads(dumps(teams))}, status_code=201)
+
+    def get_team(object_id: str) -> JSONResponse:
+        try:
+            specified_team = t_col.find_one(
+                filter={"_id": ObjectId(object_id)},
+            )
+        except (InvalidId, TypeError):
+            return JSONResponse(content={"message": "Invalid object_id format!"}, status_code=400)
+
+        if not specified_team:
+            return JSONResponse(content={"message": "The team was not found"}, status_code=404)
+
+        return JSONResponse(content={"participant": json.loads(dumps(specified_team))}, status_code=200)
```

Comment:
Are we sure there isn't a better way to handle this? If there isn't, I suggest that you export the json parsing in a utility function.

Reply:
The bson.json_util.dumps returns a JSON like string. We use the json.loads to parse the string. This is the easiest way to prevent the error ObjectID is not JSON Serializable.
Here is the result from the dumps -> 
![image](https://github.com/AUBGTheHUB/monolith/assets/38375079/6998674e-0d9d-4d26-a161-99f05870d374)
Here is the result when we parse the JSON string -> 
![image](https://github.com/AUBGTheHUB/monolith/assets/38375079/7a5e535a-4722-43d1-b15c-b648a5d043b2)